docs: inject chainable methods (#1917) (#2043)
fastify/fastify: docs/Testing.md

})
```

> you can also use chaining methods if you do not pass the callback function

Reviewer: ```suggestion
`.inject` methods can also be chained by omitting the callback function:
```

```js
fastify
  .inject()
  .get('/')
  .headers({ foo: 'bar' })
  .query({ foo: 'bar' })

>   .end((err, res) => {

Reviewer 2: ```suggestion
  .end((err, res) => { // the .end call will trigger the request
```
